### DISCUSSION TITLE
2008/07/02/%E8%A2%AB%E4%B8%AA%E5%B0%8F%E8%B4%BC%E5%90%93%E5%88%B0%E4%BA%86

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#228] [Announcements]
# 2008/07/02/%E8%A2%AB%E4%B8%AA%E5%B0%8F%E8%B4%BC%E5%90%93%E5%88%B0%E4%BA%86

昨晚又没睡好，是因为睡觉前被个小贼吓了一大跳。

https://blog.qizhen.xyz/2008/07/02/%E8%A2%AB%E4%B8%AA%E5%B0%8F%E8%B4%BC%E5%90%93%E5%88%B0%E4%BA%86

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Peter
2008年07月12日 00:00 编辑
前两年苏州这边的治安也很好，可是现在越来越不安全了。看了你的文章，刚刚把屋子里面巡视了一遍，我住在顶楼，感觉也很不安全，找出久违的水果刀防身。

回复
Nicholas
2008年07月5日 11:49 编辑
简单起见，如果移走空调就不好上的话，建议移一下空调。你住张江路外吗？貌似尤其SMIC家属区那带最不安全。。。。。。

回复
奕瑋
2008年07月5日 00:41 编辑
我一个朋友家卧室枕头下面有一个很强的电棍，门后有一棒球棍，床头柜的抽屉里还有匕首，……，临时应付小贼绰绰有余了。
建议阮主也弄个电棍给自己壮壮胆吧。

回复
WL
2008年07月4日 23:13 编辑
…..小心啊

回复
任
2008年07月3日 20:36 编辑
兵来将挡，～～～

回复
Nan
2008年07月3日 09:19 编辑
恐怖。注意安全啊。

回复
Percy
2008年07月3日 08:33 编辑
真的好可怕。。。

回复
